substrate/network/src/protocol.rs

```diff
@@ -353,7 +353,17 @@ impl<B: BlockT, S: Specialization<B>> Protocol<B, S> {
 
 	fn on_block_response(&self, io: &mut SyncIo, peer: NodeIndex, request: message::BlockRequest<B>, response: message::BlockResponse<B>) {
 		// TODO: validate response
-		trace!(target: "sync", "BlockResponse {} from {} with {} blocks", response.id, peer, response.blocks.len());
+		trace!(target: "sync", "BlockResponse {} from {} with {} blocks{}",
+			response.id, peer, response.blocks.len(),
+			match (
```

Comment:
Let's avoid such big expressions as an argument, especially when used in positional formatting.